exchanges/binance/binance.go

```diff
@@ -36,8 +36,8 @@ const (
 	ufuturesAPIURL = "https://fapi.binance.com"
 	tradeBaseURL   = "https://www.binance.com/en/"
 
-	testnetSpotURL = "https://testnet.binance.vision/api"
-	testnetFutures = "https://testnet.binancefuture.com"
+	testnetSpotURL = "https://testnet.binance.vision/api" //nolint:unused // Can be used for testing
+	testnetFutures = "https://testnet.binancefuture.com"  //nolint:unused // Can be used for testing
```

Comment:
🐒 These aren't exported .... so I think they should be comments or in exchange readme

Reply:
Okay. Now I get it.
I'm thinking they should live in `binance_live_test.go` but I'm sure you considered that.
Resolving.